src/collective/mustread/interfaces.py

```diff
@@ -190,10 +192,91 @@ def who_unread(obj, force_deadline=True):
+
+        If userid is given, limit results to objects with an open read request
+        for this userid.
+
+        @guido: if we implement the force_deadline here, we'd have an easy way
+        to list all items with missed deadlines too. alternatively we can make
+        get_report more powerful/complex
+
+        :param context: Context to search for objects with open read requests
+        :type context: Content object (must be IUUID resolvable)
+        :param userid: Userid to search for open read requests for.
+        :type userid: string
+        :returns: List of content objects with open read requests
+        :rtype: Generator
+        '''
+        # xxx guido: generator or list? (we intend to use all elements anyway)
+
+    def get_report(context=None, include_children=True, userid=None,
+                   start_date=None):
+        '''Return all mustread database entries for context.
+        (useful for exporting data or displaying detailed reports)
+
+        If context is None, defaults to plone site.
+
+        If include_children is ``True`` (which is the default), include
+        mustread records for child-objects of context.
+
+        If userid is given, only return entries for this user
+
+        If start_date is given, only return entries created after start_date
+        (``scheduled_at`` or ``read_at`` >= ``start_date``
+
+        :param context: Context to create mustread report on
+        :type context: Content object (must be IUUID resolvable)
+        :param include_children: Whether to include child-objects of context
+        :type include_children: Bool
+        :param userid: only report records for specified user
+        :type userid: string
+        :param start_date: Filter out entries older than start_date
+        :type read_at: datetime
+        :returns: List of dictionaries, each representing a mustread record
+                  ``[{userid, uid, status, read_at, deadline, scheduled_at,
+                      type, path, ...}]``
+        :rtype: Generator
+        '''
+        # xxx guido: generator or list? (we intend to use all elements anyway)
+
+    def unschedule_must_read(obj=None, userids=None):
+        '''Maintenance method to remove open must-read requests for an object
+        and/or a specified userid
+
+        @guido decide if we need/want this
+        '''
+
+    def accept_not_read(obj=None, userids=None):
+        '''Maintenance method to mark open must-read requests for an object
+        and/or a specified userid with status 'not-read' and leaving the
+        read_at date empty.
+        this way they object is not listed as what_to_read
+        and the user ist not listed in who_must_read
+
+        this would allow admins to clear certain objects form the must-read
+        list or accept that a user (that might be ill) will not be required to
+        read a content item
+
+        @guido decide if we need/want this
```

Comment:
@gyst there is a semantic difference between: delete a read request from the database (unschedule_must_read - i.e. because it was scheduled by a mistake) and accepting that a user has not confirmed an open request (accept_not_read) 

however, i also agree, that adding another possible state `accepted` (or whatever we name it) adds extra complexity and is not required for my usecase either. so i'll drop that